README.md

```diff
@@ -352,6 +353,37 @@ webgriffe_sylius_akeneo:
 
 In the `akeneo_attribute_code` option you have to set the code of the **Akeneo price attribute** where you store your products prices. Then they will be imported into Sylius for channels whose base currency is the same as the price currency on Akeneo.
 
+### Importing product metrical properties
+
+**NB. This feature is only available from Akeneo version 5**
+
+If you manage product metrical attributes on Akeneo you can import them on Sylius as product properties (like weight, length, height and depth). To do this, you have to configure the metrical properties value handler in the `config/packages/webgriffe_sylius_akeneo_plugin.yaml` file:
+
+```yaml
+# config/packages/webgriffe_sylius_akeneo_plugin.yaml
+webgriffe_sylius_akeneo:
+  # ...
+  value_handlers:
+    product:
+      # ...
+      weight:
+        type: 'metric_property'
+        options:
+          akeneo_attribute_code: 'weight'
+          sylius_property_code: 'weight'
+      height:
+        type: 'metric_property'
+        options:
+          akeneo_attribute_code: 'height'
+          sylius_property_code: 'height'
+          default_unit_measurement: 'CENTIMETER'
```

Comment:
I'd not call this `default_unit_measurement` but only `unit_measurement`. I think that the word `default` could be misunderstood as the default unit measurement of that measurement unit family and that's not the case. I'd also remove this `default` work from all the parameters and variables names used for this concept in this PR.

Reply:
At this point, I would also rename the service to UnitMeasurementValueConverter.

Reply:
I agree!